Client/Frontend/NTP/NTPLearnMoreViewController.swift

```diff
@@ -0,0 +1,150 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import Shared
+import BraveShared
+import BraveRewardsUI
+import SafariServices
+
+/// A view controller that is presented after user taps on 'Learn more' on one of `NTPNotificationViewController` views.
+class NTPLearnMoreViewController: BottomSheetViewController {
+    
+    private let state: BrandedImageCalloutState
+    
+    var linkHandler: ((URL) -> Void)?
+    
+    init(state: BrandedImageCalloutState) {
+        self.state = state
+        super.init()
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        guard let mainView = createViewFromState() else {
+            assertionFailure()
+            return
+        }
+        contentView.addSubview(mainView)
+        
+        mainView.snp.remakeConstraints {
+            $0.top.equalToSuperview().inset(28)
+            $0.left.right.equalToSuperview().inset(16)
+            $0.bottom.equalTo(view.safeAreaLayoutGuide).inset(28)
+        }
+    }
+    
+    func createViewFromState() -> NTPNotificationView? {
+        var config = NTPNotificationViewConfig(textColor: .black)
+        
+        switch state {
+        case .getPaidTurnRewardsOn:
+            config.headerText = Strings.NTP.getPaidForThisImageTurnRewards
+            
+            let tos = Strings.termsOfServiceURL
+            let tosPart = String(format: Strings.NTP.turnRewardsTos, tos)
+            
+            let hideImages = Strings.NTP.hideSponsoredImages
+            let hideImagesPart = String(format: Strings.NTP.chooseToHideSponsoredImages, hideImages)
+            
+            let fullBodyText = "\(tosPart) \(hideImagesPart)"
+            
+            config.bodyText =
+                (text: fullBodyText,
+                 urlInfo: [tos: "tos", hideImages: "hide-sponsored-images"],
+                 action: { [weak self] action in
+                    if action.absoluteString == "tos" {
+                        let urlString = "https://www.brave.com/terms_of_use"
+                        guard let url = URL(string: urlString) else { return }
+                        self?.showSFSafariViewController(url: url)
```

Comment:
Why not just open a new tab?

Reply:
because this view is UI blocking, and if it goes away user will not be able to show it anymore

it's show only once type of view, we use open in new tab for only one notification state